.ci/configure_crio_for_kata.sh

```diff
@@ -11,22 +11,9 @@ set -o pipefail
 
 crio_config_file="/etc/crio/crio.conf"
 
-# `use_runtime_class` should be set to:
-# - true if we will test using k8s RuntimeClass feature or
-# - false (default) if we will test using the old trusted/untrusted annotations.
-use_runtime_class=${use_runtime_class:-false}
-
-if [ "${use_runtime_class}"  == true ]; then
-	echo "Configure runtimes map for RuntimeClass feature"
-	echo "- Set runc as default runtime"
-	sudo sed -i 's!runtime_path =.*!runtime_path = "/usr/local/bin/crio-runc"!' "$crio_config_file"
-	echo "- Add kata-runtime to the runtimes map"
-	sudo sed -i '/crio-runc/a[crio.runtime.runtimes.kata]' "$crio_config_file"
-	sudo sed -i '/kata/aruntime_path = "/usr/local/bin/kata-runtime"' "$crio_config_file"
-else
-	echo "Configure runtimes for trusted/untrusted annotations"
```

Comment:
I think that we will still need to check for cri-o version and configure it depending the version as we still use cri-o 1.10 for openshift v3.10, which does not support runtimeclass.

Reply:
@chavafg changes applied

Reply:
@chavafg if I update the cri-o version we have a lot of failures in the cri-o tests :S

Reply:
@chavafg doing the installation of CRI-O locally with 1.14.0, I get the following error
```
Get CRI Tools
can't load package: package github.com/kubernetes-sigs/cri-tools: no Go files in /home/popular/go/src/github.com/kubernetes-sigs/cri-tools
/home/popular/go/src/github.com/kubernetes-sigs/cri-tools /home/popular/go/src/github.com/cri-o/cri-o /home/popular/go/src/github.com/kata-containers/tests/.ci
error: pathspec 'v1.14.0' did not match any file(s) known to git.
```

Reply:
but if I put the version v1.13.6 the installation is done succesfully

Reply:
Ah, thanks for the clarification on the openshift stuff @chavafg - I think that is an element I was missing in my understanding :-)